iree/compiler/Codegen/LLVMCPU/LLVMCPUTileFuseAndVectorizeLinalgTensorOps.cpp

```diff
@@ -171,10 +149,36 @@ void LLVMCPUTileFuseAndVectorizePass::runOnOperation() {
 
   // Tile and fuse for vector sizes, then tile reduction loops. We don't rely on
   // unroll vector pass because it could introduce register pressure.
+  bool hasMatmulAndIsVectorizable = true;
```

Comment:
Just an FYI, this should move into `KernelDispatch.cpp` so that we can enable this path for cases where this is true within a dispatch region.